search/utils.py

```diff
@@ -47,13 +47,28 @@ def get_conn():
     if _CONN is None:
         _CONN = connections.create_connection(hosts=[URL])
 
+    if not verify:
+        return _CONN
+
     # Make sure everything exists.
     if not _CONN.indices.exists(INDEX_NAME):
-        _CONN.indices.create(INDEX_NAME)
+        raise ReindexException("Unable to find index {index_name}".format(
+            index_name=INDEX_NAME
+        ))
+
+    mapping = _CONN.indices.get_mapping()
+    if INDEX_NAME not in mapping:
```

Comment:
This is checking for the index a second time, not the mapping. 

[This](https://github.com/mitodl/lore/pull/700#issuecomment-142340863) script contains the syntax to look `DOC_TYPE`.